### PR TITLE
fix: DBTP-1282 allow for explicit settings of PUB_PATH_LIST

### PR DESCRIPTION
**Pull request by DeveloperConnor**
Addresses [DBTP-1282](https://uktrade.atlassian.net/browse/DBTP-1282)

Previously setting a PUB_PATH_LIST explicitly to '/' would throw an error when the NGINX container spun up. See ticket for details.

Updated the entrypoint.sh for the image to allow for explicit setting of a PUB_PATH_LIST to '/'

---
### Title:
- [x] Scope included as per [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] Ticket reference included (unless it's a quick out of ticket thing)
### Description:
- [x] Link to ticket included (unless it's a quick out of ticket thing)
- [ ] Includes tests (or an explanation for why it doesn't)
- [ ] If the work includes user interface changes, before and after screenshots included in description
- [ ] Includes any applicable changes to the documentation in this code base
- [ ] Includes link(s) to any applicable changes to the documentation in the [DBT Platform Documentation](https://platform.readme.trade.gov.uk/) (can be to a pull request)
### Tasks:
- [ ] [Trigger the pull request regression tests for this branch](https://github.com/uktrade/platform-tools?tab=readme-ov-file#regression-tests) and confirm that they are passing

[DBTP-1282]: https://uktrade.atlassian.net/browse/DBTP-1282?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by DeveloperConnor** [comment]
Need to run the regression pipeline still, this runs with the below PRIV_PATH_LIST:
PRIV_PATH_LIST: /ipfilter/
So won't validate that the bugfix was successful so that will need some inc manual testing.

**Review by chiaramapellimt — APPROVED** [review]

**Review by gabelton — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -171,3 +171,14 @@ If found, it indicates that the new package version exists in PyPI.
...
+## Docker Images
+
+### Building the Images
+
+If building on a ARM mac, the image will build but will fail to deploy with the following error:
+exec /usr/bin/dumb-init: exec format error
+
+Instead, build the image via the below command, to build for the linux/amd64 platform.
+
+`DOCKER_DEFAULT_PLATFORM=linux/amd64 docker build --tag public.ecr.aws/uktrade/nginx-dbt-platform:<tag> .`
```
Comment (gabelton) [inline README.md:184]: I think either move this stuff to the [nginx readme ](https://github.com/uktrade/platform-tools/blob/ddf8085f6385a033c0235b6d4f80a2c23258ed2f/images/nginx-dbt-platform/README.md) or make the tag non-specific to nginx

Comment (DeveloperConnor) [reply README.md:184]: Will move the comment over

**Comment by DeveloperConnor** [comment]
Regression test run: https://eu-west-2.console.aws.amazon.com/codesuite/codebuild/763451185160/projects/platform-tools-test/build/platform-tools-test%3A1f540d4e-4ed4-42d9-b1de-efb0f16c8110/?region=eu-west-2. 

This was with my Nginx image, with the default of PRIV_PATH_LIST set to '/'

**Review by gabelton — APPROVED** [review]

**Pull request merged**